tests/RGB.d

```diff
@@ -0,0 +1,356 @@
+    new RGB(60f,1f,0f);
+    new RGB(60f,0.6f,0.4f);
+    new RGB(61f,0f,1f);
+    new RGB(61f,1f,1f);
+    new RGB(61f,0f,1f);
+    new RGB(61f,1f,0f);
+    new RGB(61f,0.6f,0.4f);
+    new RGB(119f,0f,1f);
+    new RGB(119f,1f,1f);
+    new RGB(119f,0f,1f);
+    new RGB(119f,1f,0f);
+    new RGB(119f,0.6f,0.4f);
+    new RGB(120f,0f,1f);
+    new RGB(120f,1f,1f);
+    new RGB(120f,0f,1f);
+    new RGB(120f,1f,0f);
+    new RGB(120f,0.6f,0.4f);
+    new RGB(121f,0f,1f);
+    new RGB(121f,1f,1f);
+    new RGB(121f,0f,1f);
+    new RGB(121f,1f,0f);
+    new RGB(121f,0.6f,0.4f);
+    new RGB(179f,0f,1f);
+    new RGB(179f,1f,1f);
+    new RGB(179f,0f,1f);
+    new RGB(179f,1f,0f);
+    new RGB(179f,0.6f,0.4f);
+    new RGB(180f,0f,1f);
+    new RGB(180f,1f,1f);
+    new RGB(180f,0f,1f);
+    new RGB(180f,1f,0f);
+    new RGB(180f,0.6f,0.4f);
+    new RGB(181f,0f,1f);
+    new RGB(181f,1f,1f);
+    new RGB(181f,0f,1f);
+    new RGB(181f,1f,0f);
+    new RGB(181f,0.6f,0.4f);
+    new RGB(239f,0f,1f);
+    new RGB(239f,1f,1f);
+    new RGB(239f,0f,1f);
+    new RGB(239f,1f,0f);
+    new RGB(239f,0.6f,0.4f);
+    new RGB(240f,0f,1f);
+    new RGB(240f,1f,1f);
+    new RGB(240f,0f,1f);
+    new RGB(240f,1f,0f);
+    new RGB(240f,0.6f,0.4f);
+    new RGB(241f,0f,1f);
+    new RGB(241f,1f,1f);
+    new RGB(241f,0f,1f);
+    new RGB(241f,1f,0f);
+    new RGB(241f,0.6f,0.4f);
+    new RGB(299f,0f,1f);
+    new RGB(299f,1f,1f);
+    new RGB(299f,0f,1f);
+    new RGB(299f,1f,0f);
+    new RGB(299f,0.6f,0.4f);
+    new RGB(300f,0f,1f);
+    new RGB(300f,1f,1f);
+    new RGB(300f,0f,1f);
+    new RGB(300f,1f,0f);
+    new RGB(300f,0.6f,0.4f);
+    new RGB(301f,0f,1f);
+    new RGB(301f,1f,1f);
+    new RGB(301f,0f,1f);
+    new RGB(301f,1f,0f);
+    new RGB(301f,0.6f,0.4f);
+    new RGB(359f,0f,1f);
+    new RGB(359f,1f,1f);
+    new RGB(359f,0f,1f);
+    new RGB(359f,1f,0f);
+    new RGB(359f,0.6f,0.4f);
+    new RGB(360f,0f,1f);
+    new RGB(360f,1f,1f);
+    new RGB(360f,0f,1f);
+    new RGB(360f,1f,0f);
+    new RGB(360f,0.6f,0.4f);
+
+    assertThrown!IllegalArgumentException(new RGB(400f, 0.5f, 0.5f),
+        "No exception thrown for hue > 360");
+
+    assertThrown!IllegalArgumentException(new RGB(-5, 0.5f, 0.5f),
+        "No exception thrown for hue < 0");
+
+    assertThrown!IllegalArgumentException(new RGB(200f, -0.5f, 0.5f),
+        "No exception thrown for saturation < 0");
+
+    assertThrown!IllegalArgumentException(new RGB(200f, 300f, 0.5f),
+        "No exception thrown for saturation > 1");
+
+    assertThrown!IllegalArgumentException(new RGB(200f, 0.5f, -0.5f),
+        "No exception thrown for brightness < 0");
+
+    assertThrown!IllegalArgumentException(new RGB(200, 0.5f, 400f),
+        "No exception thrown for brightness > 1");
+}
+
+@("test_equalsLjava_lang_Object")
+unittest
+{
+    int r = 0, g = 127, b = 254;
+    RGB rgb1 = new RGB(r, g, b);
+    RGB rgb2;
+
+    rgb2 = rgb1;
+    if (!rgb1.opEquals(rgb2)) {
```

Comment:
You can just use `==` instead of `opEquals`.

Reply:
Fixed.